jute.cpp

```diff
@@ -77,7 +77,10 @@ string jValue::to_string_d(int d) {
 	}
 	return "##";
 }
-jValue::jValue(jType tp = JUNKNOWN) {
+jValue::jValue() {
+    jType tp = JUNKNOWN;
```

Comment:
Shouldn't this be
`this->type = JUNKNOWN;`
?

Reply:
You are right, let me fixed it. Suppose default constructor never called, that's why I didn't get any errors..